package.json

```diff
@@ -65,6 +65,7 @@
     "verbose": true
   },
   "dependencies": {
-    "babel-preset-stage-2": "^6.18.0"
+    "babel-preset-stage-2": "^6.18.0",
+    "prop-types": "^15.5.6"
```

Comment:
Can you move this into devDependencies? PropTypes is only used in the tests, so it's not actually a dep.